src/classBased/components/TodoList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TodoItem from './TodoItem';
+/* eslint-disable*/
+class TodoList extends React.Component {
+  render() {
+    return (
+      <ul>
+        {this.props.todos.map((todo) => (
+          <TodoItem
+            setUpdate={this.props.setUpdate}
+            key={todo.id}
+            todo={todo}
+            handleChangeProps={this.props.handleChangeProps}
+            deleteItem={this.props.deleteItem}
```

Comment:
The delete option in class base is not working.
![image](https://user-images.githubusercontent.com/13853450/178068246-c1cae991-e3c1-49e8-964b-70a963d3d573.png)